libcbor/PKGBUILD

```diff
@@ -0,0 +1,46 @@
+# Contributer: Reza Tavakoli <rta.0070@gmail.com>
```

Comment:
This was also one of my questions, I saw both Contributer and Maintainer over different PKGBUILDs, is this the right one to pick?

Reply:
I don't know ;-)

Whatever you choose is fine with me.